Source/WebInspectorUI/UserInterface/Views/Variables.css

```diff
@@ -423,17 +423,17 @@ body {
             --border-color-secondary: hsl(0, 0%, 24%);
         }
     }
-    
-    &:is(.mac-platform):not(.mac-platform.monterey, .mac-platform.big-sur):not(.docked) {
-        /* keep in sync with `WI.undockedTitleAreaHeight` */
-        --undocked-title-area-height: calc(22px / var(--zoom-factor));
-    }
 
     &:is(.mac-platform.monterey, .mac-platform.big-sur):not(.docked) {
         /* keep in sync with `WI.undockedTitleAreaHeight` */
         --undocked-title-area-height: calc(27px / var(--zoom-factor));
     }
 
+    &:is(.mac-platform.catalina, .mac-platform.mojave):not(.docked) {
```

Comment:
support for macOS 10.14 Mojave was dropped in [242418@main](https://commits.webkit.org/242418@main)

we can/should also remove the reference to it in `WI.undockedTitleAreaHeight`

Reply:
I kept the selector for Mojave because it's still taken into account in `WI.undockedTitleAreaHeight()` but indeed it is no longer generated from `InspectorFrontendHost::platformVersionName()`&mdash;I'll remove it.

Also: How about another follow-up to remove the Mojave case in `WI.undockedTitleAreaHeight()`? 😉

Reply:
> Also: How about another follow-up to remove the Mojave case in `WI.undockedTitleAreaHeight()`? 😉

Follow-up here: https://github.com/WebKit/WebKit/pull/28331